web/src/components/FilingsCalendar.tsx

```diff
@@ -196,7 +196,12 @@ export const FilingsCalendar = (props: Props): ReactElement => {
     const type = LookupOperatingPhaseById(userData?.profileData.operatingPhase).displayCalendarType;
     if (type === "LIST") return renderCalendarAsList();
     if (type === "FULL") {
-      if (isLargeScreen && userData?.preferences.isCalendarFullView) return renderCalendarAsGrid();
+      if (
+        isLargeScreen &&
+        userData?.preferences.isCalendarFullView &&
+        userData.taxFilingData.state === "SUCCESS"
+      )
+        return renderCalendarAsGrid();
```

Comment:
@AnJuHyppolite the logic added here is not correct, I think you might need to start this one over. The story is asking us to display List View *by default* if they do not yet have a success on the tax access, and Grid View *by default* if they have a successfully accessed the tax calendar.

"By default" means the initial state the user sees. The way this is written right now, it is showing ONLY List View pre-tax-access without allowing the user to switch to List View. This story is about changing the default view ... but the user is still allowed to change their view as desired.